src/utils/soup.json

```diff
@@ -60,10 +60,7 @@
     "source_port_id": "source_port_0",
```

Comment:
we should "ignore" this file (and maybe every json file) in the `biome.json`

Reply:
I have added the `symbol_name` in this json. So, I don't think we can ignore it

Reply:
Gotcha np!